manim/mobject/geometry.py

```diff
@@ -402,9 +387,25 @@ def __init__(self, point=ORIGIN, **kwargs):
 
 
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    CONFIG = {"radius": DEFAULT_SMALL_DOT_RADIUS}
+
+
+class AnnotationDot(VMobject):
+    def __init__(self, color=BLUE):
+        VMobject.__init__(self)
+        sourunding_dot = Dot().scale(1.3).set_fill(color=WHITE).set_z_index(-1)
+        innerdot = Dot().set_color(color)
```

Comment:
Wouldn't it be better to use a `Circle`, setting its stroke and fill as you want (e.g. white and blue respectively) instead of using 2 `Dot`s and overlaying one on top of another? Are there some disadvantages of the former approach we might face?

Reply:
We should use only one Circle if at all possible like @nilaybhatia is suggesting.

Reply:
ok, i ll have a look on that

Reply:
done